MDANSE/Src/MDANSE/Framework/OutputVariables/IOutputVariable.py

```diff
@@ -61,6 +61,7 @@ def __new__(
         *,
         main_result: bool = False,
         partial_result: bool = False,
+        dtype: type = np.float64,
```

Comment:
Strictly these should be of type [`np.dtype`](https://numpy.org/doc/2.2/reference/arrays.dtypes.html)

Not sure it matters too much.